GETTING_STARTED.md

```diff
@@ -0,0 +1,148 @@
+## Basic setup
+
+This guide will take you through building your first project with Angular Material 2.  We'll be using 
+the Angular CLI, but you can also accomplish this with other build tools like Webpack or Gulp.
+
+1 - First, you'll want to globally install the CLI, if you haven't already:
+ 
+ ```bash
+ npm install -g angular-cli
+ ```
+ 
+2 - Once you have the CLI installed, you can create a new project with `ng new <project-name>`:
+ 
+ ```bash
+ ng new my-project
+ ```
+ 
+This will create a starter repo under the name you specified with the files and folders 
+you'd need for any Angular 2 app: a root component, a main.ts to bootstrap your root component, an 
+index.html, etc.  It also sets up build rules to transpile your Typescript into Javascript and sets 
+some initial config for the System.js module loader.  
+ 
+3 - Now that your project has been created, you can install any Angular Material 2 components you'd like 
+to use. You can see our [list of published packages on npm here](https://www.npmjs.com/~angular2-material).  
+
+```bash
+npm install --save @angular2-material/{core,button,card}
+```
+Note: the core module is required as a peer dependency of other components.
+
+4 - Next, you'll need to build the `@angular2-material` folder out of `node_modules` and into 
+`dist/vendor`, so that it's served with the rest of the vendor files. You can easily configure this by 
+editing the `angular-cli-build.js` file in the root of your project.  Simply add a glob for all 
+@angular2-material files to the end of the existing `vendorNpmFiles` array.
+
+**angular-cli-build.js**
+```javascript
```

Comment:
FYI, for your own future convenience, you can just type `js` or `ts` instead of `javascript` and `typescript` for GitHub markdown